gradle/plugins/aot-smoke-test-plugin/src/main/java/org/springframework/aot/gradle/AotSmokeTestPlugin.java

```diff
@@ -89,6 +91,33 @@ private void configureBootJavaProject(Project project) {
 				.mavenLocal(
 						(mavenLocal) -> mavenLocal.content((content) -> includedGroups.forEach(content::includeGroup)));
 		}
+		if (project.hasProperty("forceSnapshots")) {
+			project.getConfigurations().all((configuration) ->
+					configuration.getResolutionStrategy().eachDependency(dependency -> {
+						boolean isSnapshot = dependency.getRequested().getVersion() != null
+								&& dependency.getRequested().getVersion().endsWith("-SNAPSHOT");
+						if (dependency.getRequested().getGroup().startsWith("org.springframework") && !isSnapshot) {
```

Comment:
Should `org.springframework.boot` by skipped here? I think it probably makes sense to do so as the Boot version is already easily controlled through `gradle.properties`.